client/src/App.js

```diff
@@ -24,6 +26,8 @@ const App = () => {
 	// 					'}'
 	// 			]
 
+	// testing
```

Comment:
Remove comments unless needed, although we shouldn't still be "testing" code that is being merged to main